### DISCUSSION TITLE
List

### DISCUSSION THREAD

**Discussion by ppaxson** [ente-io/ente#3398] [Enhancements]
Hi, I'd like to request that we be able to re-order the list of one time use passcodes. I found since I switched from Google authenticator that a lot of the accounts that I don't use often are at the top and I have to dig through all of the one time use codes I don't want just to find the code I actually want. That is a feature that Google authenticator has that I would like Ente Auth to support. Otherwise, I love the app and I think you guys are doing a great job. 

Thank you.

**Comment by ua741** [comment]
Duplicate of https://github.com/ente-io/ente/discussions/497

For the time being, please use Pin functionality to move frequently used account to top.
